frontend/src/scenes/experiments/Experiments.tsx

```diff
@@ -46,27 +46,24 @@ export function Experiments(): JSX.Element {
         {
             title: 'Duration',
             render: function Render(_, experiment: Experiment) {
-                return (
-                    <div>
-                        {`${
-                            experiment.end_date
-                                ? dayjs(experiment.start_date).diff(dayjs(experiment.end_date), 'day')
```

Comment:
Was in reverse, so on calling "End", the dates were negative lol

Reply:
Looks like this now:

![image](https://user-images.githubusercontent.com/7115141/146542992-08275496-94f0-4287-b0e7-5a7c8036a8d7.png)